src/main/java/com/rackspace/ceres/app/web/DeleteMetricController.java

```diff
@@ -31,6 +32,7 @@ public DeleteMetricController(MetricDeletionService metricDeletionService) {
   @DeleteMapping
   public Mono<Empty> deleteMetrics(@RequestHeader(value = "#{appProperties.tenantHeader}") String tenantHeader,
       @RequestParam(required = false) String metricName,
+      @RequestParam(required = false) String metricGroup,
```

Comment:
I think again this should be separated out

Reply:
You mean like creating a new endpoint for deletion by metric group?

Reply:
yes